src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/generalapplicationdirections/GeneralApplicationDirectionsNewEventAboutToSubmitHandler.java

```diff
@@ -74,17 +79,17 @@ public GenericAboutToStartOrSubmitCallbackResponse<FinremCaseData> handle(
 
         log.info(CallbackHandlerLogger.aboutToSubmit(callbackRequest));
         FinremCaseDetails caseDetails = callbackRequest.getCaseDetails();
+
         FinremCaseData caseData = caseDetails.getData();
+        helper.populateGeneralApplicationSender(caseData,
+            caseData.getGeneralApplicationWrapper().getGeneralApplications());
 
         //Invoke performAddHearing when hearing is required
```

Comment:
Move line 88 to 91 to a private method to enhance readability. e.g. `performAddHearingIfNecessary`